test-assets/test-projects/NUnitTestProject/NuGet.config

```diff
@@ -0,0 +1,9 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+    <add key="AppVeyor NUnit CI Feed" value="https://ci.appveyor.com/nuget/nunit" />
```

Comment:
As soon as there's an NUnitTestAdapter on nuget.org that we can use, I'll just delete this file.